core/src/main/java/bisq/core/dao/state/storage/BsqBlocksStorageService.java

```diff
@@ -123,9 +123,11 @@ void copyFromResources(String postFix) {
             }
             for (String fileName : fileNames) {
                 File destinationFile = new File(storageDir, fileName);
-                FileUtil.resourceToFile(resourceDir + "/" + fileName, destinationFile);
+                FileUtil.resourceToFile(resourceDir + File.separator + fileName, destinationFile);
```

Comment:
This doesn't appear to work on Windows. It has to be "/", not "\\".

Reply:
Ah really? Weird... Will revert that. Javas resource handling is really a pain.